libs/packages/components/src/lib/autocomplete-search/autocomplete-search.component.html

```diff
@@ -41,7 +41,45 @@
         class="usa-list usa-list--unstyled sds-autocomplete"
         (scroll)="onScroll()"
       >
-        <ng-container *ngIf="results && results.length">
+        <ng-container *ngIf="configuration.useCheckBoxes">
+          <label
```

Comment:
I'm seeing a cursor when mousing over this section, but clicking does not seem to do anything, can the pointer be updated when mousing over this section to not give the visual indicator that clicking will result in some action?

Reply:
will update this sam-styles

Reply:
done